.github/workflows/migrator.yml

```diff
@@ -436,16 +436,31 @@ jobs:
               owner: 'ansys',
               repo: 'pymapdl',
               issue_number: prNumber,
-              body: `**:rocket: Migration completed!**
+              body: `## :rocket: Migration completed!
 
             The PR [#${newPrData.data.number}](${newPrData.data.html_url}) has been created successfully.
 
-            Thank you @${userTriggering} for your contribution! Please review the PR and make any necessary changes.`,
+            Thank you @${userTriggering} for your contribution! Please review the PR and make any necessary changes.
+
+            You can clone this branch in a new repository by doing:
+
+            ~~~terminal
+            git clone -b ${prHeadBranch} --single-branch https://github.com/ansys/pymapdl.git
```

Comment:
Can't you programatically get the repository from the PR context? Just wondering

Reply:
Yes, I can. I believe `github.repository` is the way.